### ISSUE TITLE
Error: estimates.json: missing field `Mean`

### ISSUE THREAD

**Issue by sharksforarms** [boa-dev/criterion-compare-action#12]
I've been getting this error, not really sure why.

Workflow:
https://github.com/sharksforarms/rust-packet/blob/master/.github/workflows/pull_request.yml

CI:
https://github.com/sharksforarms/rust-packet/pull/25/checks?check_run_id=865820168

```
/usr/share/rust/.cargo/bin/critcmp master changes
/home/runner/work/rust-packet/rust-packet/target/criterion/bench_raw_from_bytes/new: estimates.json: missing field `Mean` at line 1 column 984
Error: The process '/usr/share/rust/.cargo/bin/critcmp' failed with exit code 1
    at ExecState._setResult (/home/runner/work/_actions/boa-dev/criterion-compare-action/master/dist/index.js:1:12035)
    at ExecState.CheckComplete (/home/runner/work/_actions/boa-dev/criterion-compare-action/master/dist/index.js:1:11597)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/boa-dev/criterion-compare-action/master/dist/index.js:1:10445)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:430:11)
    at Socket.emit (events.js:210:5)
    at Pipe.<anonymous> (net.js:659:12)
##[error]Unhanded error:
Error: The process '/usr/share/rust/.cargo/bin/critcmp' failed with exit code 1
```

**Comment by Razican** [comment]
Hi, this is a known issue in `critcmp` since `criterion` released the 0.3.3 version: https://github.com/BurntSushi/critcmp/issues/5

The workaround for this is to force criterion version 0.3.2 in your project by using `"=0.3.2" as the criterion version.

**Comment by Razican** [comment]
This has been fixed now.
